server/src/main/java/org/eclipse/lsp/cobol/core/model/tree/variables/VariableDefinitionUtil.java

```diff
@@ -62,9 +69,80 @@
   public static final String REDEFINE_IMMEDIATELY_FOLLOW = "semantics.redefineImmediatelyFollow";
   public static final String LEVELS_MUST_MATCH = "semantics.levelsMustMatch";
   public static final String REDEFINED_CONTAIN_VALUE = "semantics.redefinedContainValue";
+  private static final String PIC_AND_USAGE_CONFLICT = "semantics.picAndUsageConflict";
 
   public static final ErrorSeverity SEVERITY = ERROR;
 
+  static final List<UsageFormat> BLANK_AND_SIGN_NOT_ALLOWED_USAGE =
+          ImmutableList.of(UTF_8, NATIONAL);
+
+
+  static final ImmutableList<UsageFormat> STRING_USAGE_FORMAT =
```

Comment:
This class contains too much logic, what about extracting a `UsageFormatUtils`?

Reply:
Done